test/images/volumes-tester/iscsi/create_block.sh

```diff
@@ -33,8 +33,9 @@ cleanup()
 
 trap cleanup TERM EXIT
 
-# Create 1MB device with ext2
-dd if=/dev/zero of=block count=1 bs=1M
+# Create 120MB device with ext2
+# (volume_io tests need at least 100MB)
+dd if=/dev/zero of=block seek=120 count=1 bs=1M
 mkfs.ext2 block
```

Comment:
While we're touching this script, what do you think about using ext4? Its much more common than ext2.

Reply:
ext2 fits 1MB image well. Now that we have 120MB, I don't know. Would be the image (in tar.gz format) significantly bigger?